Library/Homebrew/attestation.rb

```diff
@@ -179,6 +179,8 @@ def self.check_attestation(bottle, signing_repo, signing_workflow = nil, subject
       attestation
     end
 
+    ATTESTATION_MAX_RETRIES = 5
```

Comment:
```suggestion
    ATTESTATION_MAX_RETRIES = 2
```
or 
```suggestion
    ATTESTATION_MAX_RETRIES = 3
```
would be fine with me if you want to fail earlier